sdk/python/kfp/client/client.py

```diff
@@ -0,0 +1,1555 @@
+        if cookies is None:
+            cookies = self._context_setting.get('client_authentication_cookie')
+        api_client = kfp_server_api.api_client.ApiClient(
+            config,
+            cookie=cookies,
+            header_name=self._context_setting.get(
+                'client_authentication_header_name'),
+            header_value=self._context_setting.get(
+                'client_authentication_header_value'))
+        _add_generated_apis(self, kfp_server_api, api_client)
+        self._job_api = kfp_server_api.api.job_service_api.JobServiceApi(
+            api_client)
+        self._run_api = kfp_server_api.api.run_service_api.RunServiceApi(
+            api_client)
+        self._experiment_api = kfp_server_api.api.experiment_service_api.ExperimentServiceApi(
+            api_client)
+        self._pipelines_api = kfp_server_api.api.pipeline_service_api.PipelineServiceApi(
+            api_client)
+        self._upload_api = kfp_server_api.api.PipelineUploadServiceApi(
+            api_client)
+        self._healthz_api = kfp_server_api.api.healthz_service_api.HealthzServiceApi(
+            api_client)
+        if not self._context_setting['namespace'] and self.get_kfp_healthz(
+        ).multi_user is True:
+            try:
+                with open(Client.NAMESPACE_PATH, 'r') as f:
+                    current_namespace = f.read()
+                    self.set_user_namespace(current_namespace)
+            except FileNotFoundError:
+                logging.info(
+                    'Failed to automatically set namespace.', exc_info=False)
+
+    def _load_config(self, host, client_id, namespace, other_client_id,
+                     other_client_secret, existing_token, proxy, ssl_ca_cert,
+                     kube_context, credentials, verify_ssl):
+        config = kfp_server_api.configuration.Configuration()
+
+        if proxy:
+            # https://github.com/kubeflow/pipelines/blob/c6ac5e0b1fd991e19e96419f0f508ec0a4217c29/backend/api/python_http_client/kfp_server_api/rest.py#L100
+            config.proxy = proxy
+        if verify_ssl is not None:
+            config.verify_ssl = verify_ssl
+
+        if ssl_ca_cert:
+            config.ssl_ca_cert = ssl_ca_cert
+
+        host = host or ''
+
+        # Defaults to 'https' if host does not contain 'http' or 'https' protocol.
+        if host and not host.startswith('http'):
+            warnings.warn(
+                'The host %s does not contain the "http" or "https" protocol.'
+                ' Defaults to "https".' % host)
+            host = 'https://' + host
+
+        # Preprocess the host endpoint to prevent some common user mistakes.
+        if not client_id:
+            # always preserving the protocol (http://localhost requires it)
+            host = host.rstrip('/')
+
+        if host:
+            config.host = host
+
+        token = None
+
+        # "existing_token" is designed to accept token generated outside of SDK.
```

Comment:
What do you think about removing this commented out code? I don't know much about its origin, but we're not using it here nor in deprecated.

Reply:
Not knowing much about the origin either, but `existing_token` seems to me it's still used as an optional parameter.
Maybe this chunk of comments should really be moved into the docstring instead.